key/group.go

```diff
@@ -201,6 +207,10 @@ func (g *Group) FromTOML(i interface{}) (err error) {
 	if err != nil {
 		return err
 	}
+	g.CatchupPeriod, err = time.ParseDuration(gt.CatchupPeriod)
+	if err != nil {
+		log.DefaultLogger().Warn("group_toml", "failed to parse catchup period", "err", err, "CatchupPeriod", gt.CatchupPeriod)
```

Comment:
If catchup period is set (i.e. is different than "") then if it is an invalid value it should return an error - no need to log.